Products/PleiadesEntity/browser/adapters/place.py

```diff
@@ -17,10 +17,10 @@
 class PlaceExportAdapter(WorkExportAdapter, ContentExportAdapter):
 
     def _connectsWith(self):
-        return self.brain.connectsWith or []
+        return getattr(self.brain, 'connectsWith', None) or []
 
     def _hasConnectionsWith(self):
-        return self.brain.hasConnectionsWith or []
+        return getattr(self.brain, 'hasConnectionsWith', None) or []
```

Comment:
I was also missing some metadata columns in my local instance. I updated catalog.xml to include them, so if you add the ones that are there but not in your catalog you should be all set.